tests/acceptance/expected-failures-on-OC-storage.txt

```diff
@@ -500,13 +496,6 @@ apiTrashbin/trashbinFilesFolders.feature:241
 apiTrashbin/trashbinFilesFolders.feature:242
 apiTrashbin/trashbinFilesFolders.feature:255
 apiTrashbin/trashbinFilesFolders.feature:256
-apiTrashbin/trashbinFilesFolders.feature:273
-apiTrashbin/trashbinFilesFolders.feature:274
-apiTrashbin/trashbinFilesFolders.feature:275
-apiTrashbin/trashbinFilesFolders.feature:276
-apiTrashbin/trashbinFilesFolders.feature:277
-apiTrashbin/trashbinFilesFolders.feature:278
-apiTrashbin/trashbinFilesFolders.feature:289
```

Comment:
These are Scenario Outline Examples for the "new" dav-path. The Examples for the "old" dav-path were already passing. So the "rewiring" in this PR has fixed some issue with the "new" dav-path. That is "a good thing" (tm)